apps/web/pages/index.vue

```diff
@@ -144,6 +146,7 @@ watch(
   },
   { immediate: true },
 );
+const shouldDisplayComponent = computed(() => runtimeConfig.public.newsletterForm).value;
```

Comment:
```suggestion
const displayNewsletter = ref(runtimeConfig.public.newsletterForm);
```

Reply:
Question: Why do you want to change to ref ?